addons/l10n_ie/data/template/account.group-ie.csv

```diff
@@ -0,0 +1,69 @@
+"id","code_prefix_start","code_prefix_end","name"
+"l10n_ie_account_group_1","1","","FIXED ASSETS"
```

Comment:
to capsloco or not to capsloco

Reply:
Does that really bother you or trolling :smile:  ?

Reply:
I really think we should abolish informatics-Medieval practices if we can.  It could be officially required of course.

Reply:
It's not, Ireland has no official CoA. But main account group are made like this in a lot of localization.
I'm fine changing that tho :+1: